gobblin-runtime/src/test/java/gobblin/runtime/local/LocalJobLauncherTest.java

```diff
@@ -41,7 +41,6 @@ public void startUp()
     this.properties = new Properties();
     this.properties.load(new FileReader("gobblin-test/resource/gobblin.test.properties"));
     this.properties.setProperty(ConfigurationKeys.JOB_HISTORY_STORE_ENABLED_KEY, "true");
-    this.properties.setProperty(ConfigurationKeys.METRICS_ENABLED_KEY, "true");
```

Comment:
Originally I thought JobMetrics would report metrics to metric log files once metric collection and reporting are enabled. I thought this would be overdo for unit tests. Then I just realized that we have a separate config key `metrics.reporting.file.enabled` to control whether to report to metric log files. I will turn that off instead.